packages/bottender-dialogflow/README.md

```diff
@@ -38,7 +38,7 @@ const Dialogflow = dialogflow({
 
 module.exports = async function App() {
   return chain([
-    Dialogflow, //
+    Dialogflow,
```

Comment:
Actually, I use it to prevent prettier formatting those lines to:

```js
module.exports = async function App() {
  return chain([Dialogflow, Unknown])
};
```

But maybe we can have a better solution.

Reply:
oh ok, let us think about it 😃

Reply:
Maybe write some comment here?
ex: trigger dialogflow actions